src/components/panel/panel.js

```diff
@@ -865,15 +875,15 @@ MdPanelRef.prototype.close = function() {
         return self.detach();
       }, this._$q.reject)
       .then(function() {
-        // TODO(ErinCoughlan) - Add destroy. This will make the code here
-        // different than just calling this.detach().
+        return self.destroy();
```

Comment:
Very good point, initially I thought this was always going to get called on close so I followed that model when refactoring.

However, I believe the user probably doesn't always want to destroy the panel, so I'm just going to make it a stand alone function, which is great because that makes the code easier!